frontend/src/app/commonComponents/QueryWrapper.tsx

```diff
@@ -1,9 +1,20 @@
 import React from "react";
 import { gql, QueryHookOptions, useQuery } from "@apollo/client";
+import {
+  useAppInsightsContext,
+  useTrackEvent,
+} from "@microsoft/applicationinsights-react-js";
+
+export type InjectedQueryWrapperProps =
+  | "data"
+  | "trackAction"
+  | "refetch"
+  | "startPolling"
+  | "stopPolling";
 
 const defaultQueryOptions: QueryHookOptions = {
   variables: {},
-  fetchPolicy: "no-cache",
+  fetchPolicy: "network-only",
```

Comment:
Is this change relatively safe if we push it to Prod quickly?

Reply:
I went back to `no-cache` to be completely sure.